asistente_ladm_col/gui/create_administrative_source_cadastre_wizard.py

```diff
@@ -52,15 +53,27 @@ def __init__(self, iface, db, qgis_utils, parent=None):
         self.mMapLayerComboBox.setFilters(QgsMapLayerProxyModel.NoGeometry)
 
     def adjust_page_1_controls(self):
+
+        combox_data = self.qgis_utils.fields_mapping(ADMINISTRATIVE_SOURCE_TABLE)
```

Comment:
We don't really need the combox_data variable, do we?

Reply:
yes, we don't need the combox_data variable